subsys/metrics/src/main/resources/default-metrics.conf

```diff
@@ -0,0 +1,8 @@
+[metrics]
```

Comment:
I think for the default metrics, it will look the same as src/main/conf/conf.d/metrics.conf (see my comment above). It seems strange, but this will get packaged inside the jar, and read from the classpath in case the user doesn't supply a configuration file with the `[metrics]` header.